test/includes/scopetest.jl

```diff
@@ -0,0 +1,5 @@
+@testset "Scoping Tests" begin
+    @test_throws LoadError @nbinclude("scoping.ipynb")
+    @nbinclude("scoping.ipynb"; softscope = true)
+    @test a == 11
+end 
```

Comment:
I would put this test directly into runtests.jl

Reply:
Done.